### DISCUSSION TITLE
Regarding the issue of recreating in the Chinese version of Minecraft

### DISCUSSION THREAD

**Discussion by 2639823773** [vectorwing/FarmersDelight#935] [General]
Hello Vectorwing,

I am a developer for Minecraft China Edition and have used NetEase's Bedrock Edition API to mostly recreate the "Farmer's Delight" mod, which I have published on NetEase's community. However, many players believe that I have directly used your code. In reality, the development processes for Bedrock Edition and Java Edition are different, and as a developer, you should understand this.

Moreover, the main package of this mod is free. I plan to continue developing other add-ons for "Farmer's Delight" in the future, but these add-ons will be free for a week and then paid. This is because offering all add-ons for free cannot sustain the continued development of this series.

I am curious about your opinion on this matter. If I have offended you in any way, I will stop developing the "Farmer's Delight" series of mods.

**Comment by RaymondBlaze** [comment]
The MIT License Farmer's Delight used applies to the whole repository and mod, thus including the assets (textures, models and etc.).
If you've used these assets in your bedrock addon project, you need to follow the terms of the MIT License, that is, to include the MIT License file (you may find it in this repository) in your addon file and provide proper credit to the original copyright holder (vectorwing).
As for the FD addons, do note that a lot of FD addons for JE uses a different license variating from ARR to LGPL, please respect their license if you wish to use their assets in your work (typically, ARR and LGPL is incompatible with your usecase anyways). Vectorwing is not capable of giving you any permission on those addons due to not being the copyright holder.

**Reply by 2639823773** [reply]
Understood, thank you.

**Comment by vectorwing** [comment]
Hello! Sorry I couldn't get to you before, I was on an extended break.

I had previously discussed this topic on a different thread here, but didn't realize it referred to a Bedrock Edition port. You are allowed to try your hand at this kind of port for sure, so long as you abide by the MIT License. If you are developing the port on your own accord, it's fine.

However, I would prefer that FD isn't monetized by third-party porting teams. If this monetization is present only within your own original add-ons, that's fine; just make sure the actual port of FD remains free.

**Reply by 2639823773** [reply]
Alright, thanks for the reply!
